README.md

```diff
@@ -15,15 +15,15 @@ technologies the FreeBSD operating system has to offer. It is geared for ease
  of use with a simple and easy to understand command syntax.
 
 iocage is in the FreeBSD ports tree as sysutils/py-iocage.
-To install using binary packages, simply run: `pkg install py36-iocage`
+To install using binary packages, simply run: `pkg install py37-iocage`
 
 ## Installation
 
 ### GitHub:
 
 The FreeBSD source tree ***must*** be located at `$SRC_BASE` (`/usr/src` by default) to build from git.
 
-- `pkg install python36 git-lite py36-cython py36-pip`
+- `pkg install python37 git-lite py37-cython py37-pip`
```

Comment:
Maybe add package `py37-libzfs` here, instead of it being obscured in the Makefile